Docs/src/tutorials/basictutorials/basictutorial1.md

```diff
@@ -0,0 +1,347 @@
+```
+Details about camera will be covered in the following tutorial.
+
+The next thing we do is ask the SceneManager to create an Entity.
+
+```cpp
+Ogre::Entity* ogreEntity = scnMgr->createEntity("ogrehead.mesh");
+```
+The parameter given to this function must be a mesh that was loaded by Ogre's resource manager. For now, resource loading is one of the many things that BaseApplication is taking care of for us. It will be explained further in later tutorials.
+
+Now that we have an Entity, we need to create a SceneNode so the Entity can be displayed in our scene. Every SceneManager has a root node. That node has a method called createChildSceneNode that will return a new SceneNode attached to the root. In older versions of Ogre, you were required to provide a unique name for your Entities and SceneNodes. This is now optional. Ogre will generate names for them if you do not provide one.
+
+```cpp
+Ogre::SceneNode* ogreNode = scnMgr->getRootSceneNode()->createChildSceneNode();
+```
+
+We save the SceneNode pointer that is returned by the method so that we can attach our Entity to it.
+
+```cpp
+ogreNode->attachObject(ogreEntity);
+```
+
+We now have a basic scene set up. Compile and run your application. You should see an Ogre's head on your screen. This is only the beginning...
+
+![](bt1_first_run.png)
+
+## Coordinates Systems
+
+Before we go on, let's cover some basics of Ogre's coordinate system. Ogre, like many other graphics engines, uses the x-z plane as the "floor" in a scene. This means that the y-axis is the vertical axis to ensure Ogre is using a [right-handed coordinate system](http://mathworld.wolfram.com/Right-HandedCoordinateSystem.html)
+
+![](bt1_display1921.png)
+
+The x-axis starts with negative values to the left and increases to the right (passing through zero at the origin). The z-axis runs forwards and backwards. The positive direction of the z-axis points "out of the screen". So if a character walks towards the screen, then its z value will be increasing. Finally, the y-axis runs from the bottom to the top. Values that are "below ground" are negative. Don't take these terms in parenthesis literally. You can put the ground wherever you want. It is just to help you orient yourself in the scene.
+
+When you run your application, notice how your Ogre head is facing towards the camera down the positive z-axis. This is a property of the mesh itself and the orientation of the camera. Cameras are covered in a later tutorial. The Ogre head is sitting at the origin of our world, (0, 0, 0). The direction the head is facing by default is a result of which way it was facing when it was originally modeled. You can effectively change this from within Ogre as well, but it will require some knowledge of quaternions, which aren't really covered until the [Intermediate Tutorials](http://www.ogre3d.org/tikiwiki/tiki-index.php?page=Intermediate+Tutorials).
+
+[//]: <> (TODO: Replace link with manual page when time has come)
```

Comment:
better put a dummy link like [this] (#) instead of linking outdated pages.

Reply:
Basically we just left stubs until we add some Intermediate tutorials. Right?

Reply:
yes